neo/Prompt/Commands/Wallet.py

```diff
@@ -147,11 +145,9 @@ def execute(self, arguments):
                 return PromptData.Wallet
             except Exception as e:
                 print("Could not open wallet: %s" % e)
-                return
 
         else:
             print("Please specify a path")
-            return
```

Comment:
Yeah I notified it here: https://github.com/CityOfZion/neo-python/pull/733#discussion_r238490876
I think `construct_send_basic` check should stay as is because it can be called from somewhere else.

Reply:
Ooh. 😊  Sorry, I didn't see that earlier. Okay.